tools/api-markdown-documenter/api-report/api-markdown-documenter.alpha.api.md

```diff
@@ -100,7 +100,7 @@ export interface ApiItemTransformations {
     // (undocumented)
     readonly [ApiItemKind.PropertySignature]: TransformApiItemWithoutChildren<ApiPropertySignature>;
     // (undocumented)
-    readonly [ApiItemKind.TypeAlias]: TransformApiItemWithoutChildren<ApiTypeAlias>;
+    readonly [ApiItemKind.TypeAlias]: TransformApiItemWithChildren<ApiTypeAlias>;
```

Comment:
Unclear if this was initially an oversight, or an observation of what is almost certainly an API-Extractor bug. TypeAlias items, for whatever reason, do not get docs generated for child "members". I will be filing a bug for this against API-Extractor. But assuming this underlying issue is fixed, we _should_ render details for TypeAlias items' children.

Reply:
How would a type alias with children look like?

Reply:
E.g.

```typescript
export type Foo = {
    bar: string;
};
```

I would expect a child property member called "bar" to be a child ("members") property under the `ApiTypeAlias` item generated for `Foo`. But members are _never_ generated for such items.

Reply:
I suspect this is a design oversight stemming from what is presumably the more common use case for type aliases (over interfaces), e.g.:

```typescript
export type Foo = Bar | Baz;
```

or 
```typescript
export type Foo = Bar & Baz;
```

Reply:
Oh, I thought type alias referred specifically to things like `export type MyAlias = string;`. Any kind of type declaration is a `TypeAlias` then? Makes more sense.

Reply:
It is, yes. I think at the AST level, there isn't a distinction between the syntax you referenced and the initial example I shared. I _think_ my example is equivalent to defining a type alias for an anonymous type. That also allows them to handle cases like:

```ts
export type Foo = Bar & {
    baz: string;
};
```